codacy/values-production.yaml

```diff
@@ -363,3 +292,69 @@ minio:
   persistence:
     enabled: true
     size: 20Gi
+
+github: {}
```

Comment:
This block is missing some indentation I believe:

```suggestion
  github: {}
```

Reply:
It was properly indented, but the following ones were not. 👍